arch/arm/boot/dts/qcom/msm8939.dtsi

```diff
@@ -27,7 +27,7 @@
 	qcom,msm-id = <239 0>, <241 0>, <263 0>;
 
 	chosen {
-		bootargs = "boot_cpus=0,4,5,6,7 sched_enable_hmp=1";
+		bootargs = "boot_cpus=0,1,2,3,4,5,6,7 sched_enable_hmp=1";
```

Comment:
sched_enable_hmp param was added here: https://github.com/ferrari-dev/android_kernel_xiaomi_ferrari/commit/c1f998027edb0b83b067fb29bd1b68417ccfd127

it seems to be a big-little thing.